extensions/azurecore/src/azureResource/providers/databaseServer/databaseServerService.ts

```diff
@@ -5,7 +5,7 @@
 
 
 import { ResourceServiceBase, GraphData } from '../resourceTreeDataProviderBase';
-import { AzureResourceDatabaseServer } from '../../interfaces';
+import { azureResource } from '../../azure-resource';
```

Comment:
Even though pre-existing the casing of interface name azureResource is inconsistent with interface names we have used elsewhere in our codebase which all begin with a capital letter.

Reply:
azureResource is a namespace here - not an interface